internal/cli/cli.go

```diff
@@ -106,6 +116,24 @@ func (c *RootCommand) runE(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	loginKey := getJWTClaimKey(viper.GetString("jwt-claim-login"))
```

Comment:
nit: This is not really a getter.  Its testing if the value is valid.   `isValidClaimKey(key) error` is more clear.

Reply:
done